### DISCUSSION TITLE
Improvement Request: OVN integration

### DISCUSSION THREAD

**Discussion by wangwei0537** [apache/cloudstack#9845] [Q&A] [answered]
Does the author have any plans to integrate OVN into CloudStack?

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template!

**Comment by kiranchavala** [comment, marked as answer]
@wangwei0537 

Currently their is  ovs intergration with cloudstack 

https://docs.cloudstack.apache.org/en/latest/plugins/ovs-plugin.html

https://www.shapeblue.com/networking-kvm-for-cloudstack/

**Comment by wangwei0537** [comment]
thanks
